backend/readers_backend/core/urls.py

```diff
@@ -18,15 +18,18 @@
 from rest_framework.authtoken.views import obtain_auth_token
 from rest_framework.routers import DefaultRouter
 from core.views import FeedbackView, DestroyAuthTokenView, CollectionViewSet, WorkbookViewSet
+from django.contrib import admin
 
 router = DefaultRouter()
-router.register('collections', CollectionViewSet, basename='collection')
-router.register('workbooks', WorkbookViewSet, basename='workbook')
+router.register('api/collections', CollectionViewSet, basename='collection')
+router.register('api/workbooks', WorkbookViewSet, basename='workbook')
+
 
 urlpatterns = [
-    path("token/", obtain_auth_token, name="token-obtain"),
-    path("token/destroy/", DestroyAuthTokenView.as_view(), name="token-destroy"),
-    path("feedback/", FeedbackView.as_view(), name="feedback"),
+    path("api/token/", obtain_auth_token, name="token-obtain"),
```

Comment:
Does Django offer some sort of context manager that might let you syntactically factor out and nest the repeated api bits of these paths?